src/group.ts

```diff
@@ -339,50 +340,58 @@ export class Group {
         function cmov(x: FieldElt, y: FieldElt, b: boolean): FieldElt {
             return b ? y : x
         }
+        // Input: u and v, elements of F, where v != 0.
+        // Output: (isQR, root), where
+        //   isQR = True  and root = sqrt(u / v) if (u / v) is square in F, and
+        //   isQR = False and root = sqrt(Z * (u / v)) otherwise.
```

Comment:
Matches steps from https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-hash-to-curve-14#appendix-F.2.1.2.

Reply:
done